src/HtmlUtils.js

```diff
@@ -63,10 +64,19 @@ const PERMITTED_URL_SCHEMES = ['http', 'https', 'ftp', 'mailto', 'magnet'];
  * need emojification.
  * unicodeToImage uses this function.
  */
-export function containsEmoji(str) {
+export function mayContainEmoji(str) {
```

Comment:
not sure this new name is any clearer - `may` implies permission, and the jsdoc says it's fairly certain on the results it provides. Maybe `mightContainEmoji` for a more accurate name?

Reply:
Ah fair enough! `might` seems to convey the meaning better, will change.